app/models/wiki.rb

```diff
@@ -101,7 +96,7 @@ def self.default_wiki
   end
 
   def self.get_or_create(language:, project:)
-    language = nil if project == 'wikidata'
+    language = nil if MULTILINGUAL_PROJECTS.include?(project)
```

Comment:
ok, I probably misunderstood your advice earlier. I'll take a look tomorrow when I can get back on this.